app/components/channel_drawer/channels_list/channel_unread_item/channel_unread_item.js

```diff
@@ -10,45 +10,44 @@ import {
 } from 'react-native';
 
 import Badge from 'app/components/badge';
-import ChanneIcon from 'app/components/channel_icon';
-import {preventDoubleTap} from 'app/utils/tap';
+import ChannelIcon from 'app/components/channel_icon';
+import {wrapWithPreventDoubleTap} from 'app/utils/tap';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
-export default class ChannelItem extends PureComponent {
+export default class ChannelUnreadItem extends PureComponent {
```

Comment:
Why have two near-identical components for this? If you're concerned about having non-unread channels spending time getting their channel member, you could use a prop (like hasUnread) to control whether or not that happens

Reply:
Yeah maybe I can take care of that, let me take a look at what can I do here